tests/projects/Sample_VS2013_FSharp_ConsoleApp_net40/Program.fs

```diff
@@ -1,7 +1,7 @@
-﻿// Learn more about F# at http://fsharp.org
+﻿// Learn more about F# at https://fsharp.org
```

Comment:
Is there any actual benefits in updating the link in test data?

Reply:
not in the test data. In several other files are links to fsharp.org (I just found 87 references) so I decided to do a global search&replace for http://fsharp.org which included also the test data. Should do any harm there IMHO.